core/templates/dev/head/pages/exploration-editor-page/feedback-tab/services/thread-data.service.ts

```diff
@@ -103,7 +103,8 @@ angular.module('oppia').factory('ThreadDataService', [
             }
           }
         }
-      }).then(onSuccess);
+        return _data;
```

Comment:
We'd need to drop onSuccess as an arg, right?

Reply:
Done.